libuavcan/include/uavcan/build_config.hpp

```diff
@@ -151,6 +151,13 @@
 # define UAVCAN_USE_EXTERNAL_FLOAT16_CONVERSION 0
 #endif
 
+/*
+ * Disables timing-dependent tests in the test suite, since these may fail on virtualized machines
+ */
+#ifndef UAVCAN_NO_TIMING_SENSITIVE_TESTS
+# define UAVCAN_NO_TIMING_SENSITIVE_TESTS 0
```

Comment:
This option is development-specific, so it's better to keep it away from `build_config.hpp`. How about this solution: http://stackoverflow.com/a/7208148/1007777 - it is not intrusive to the library. We could also apply some naming pattern for time-sensitive tests, e.g. suffix them with `RealTime`, and then disable with `--gtest_filter=-*RealTime`.